### ISSUE TITLE
Exits immediately when executed

### ISSUE THREAD

**Issue by lovelaced** [tmathmeyer/tint3#19]
I attempted installing it through the AUR as well as pulling it from github. Both created a process and then gave exit status 1 immediately.

**Comment by tmathmeyer** [comment]
can you post your ~/.tint3rc?

**Comment by tmathmeyer** [comment]
I think the problem might be occurring for you if you have not customized your ~/.tint3rc file.  I've pushed a patch, as well as updated the AUR package. If it is still failing, can you add a line "options debug" to your ~/.tint3rc file in the [[bar]] section, and report the output back?

**Comment by lovelaced** [comment]
I think the issue was that it wasn't creating a default ~/.tint3rc, and I had tried creating one in ~/.config/tint3/tint3rc but that didn't work either (I attempted using the default config found in the source).

I reinstalled from the AUR just now and it did the same thing - it doesn't seem to be writing the default config initially. Creating a .tint3rc file with the example config in the repo made it work though, even if I thereafter remove those files. I then uninstalled and reinstalled (after removing the config files) and it seemed to work fine out of the box.
